src/sardana/tango/macroserver/MacroServer.py

```diff
@@ -397,7 +397,17 @@ class MacroServerClass(SardanaDeviceClass):
              "basis. Backwards incompatible changes (up to and including "
              "its removal) may occur if deemed necessary by the "
              "core developers.",
-             None]
+             None],
+        'LogstashCacheDbPath':
+            [DevString,
+             "Path to the Logstash cache database [default: None]. "
+             "It is advised not to use the database cache, as it may "
+             "have negative effects on logging performance. See #895. "
+             "This property has been included in Sardana on a provisional "
+             "basis. Backwards incompatible changes (up to and including "
+             "its removal) may occur if deemed necessary by the "
+             "core developers.",
+             "/tmp/sardana-ms-logstash-cache.db"]
```

Comment:
I think here is the default value? Isn't it?

Reply:
Right, I removed it from the code, but forgot here.